Cargo.toml

```diff
@@ -82,8 +82,3 @@ members = [
 
 
 ]
-
-[patch.crates-io]
-gotham = { path = "gotham" }
-gotham_derive = { path = "gotham_derive" }
-borrow-bag = { path = "misc/borrow_bag" }
```

Comment:
Could you restore this, please? For development, it's certainly easier to use these together, rather than publishing individually.

Reply:
This is not needed anymore because Cargo supports both `path` and `version` in `dependencies` which I added in a previous PR. I missed one spot and I fixed it in this PR.

Reply:
And since cargo doesn't support publishing all packages at once, you can use https://crates.io/crates/cargo-workspaces to do it which works with `path` and `version` in `dependencies` without this being needed.

Reply:
@nyarly Can you please reply to this comment? I want to get this PR resolved.